.github/workflows/cd-dgraph.yml

```diff
@@ -0,0 +1,84 @@
+        run: |
+          #!/bin/bash
+          BADGER_RELEASE_VERSION=$(cat go.mod | grep -i "github.com/dgraph-io/badger" | awk '{print $2}')
+          echo "setting badger version "$BADGER_RELEASE_VERSION
+          echo "BADGER_RELEASE_VERSION=$BADGER_RELEASE_VERSION" >> $GITHUB_ENV
+      - name: Download Badger Artifacts
+        run: |
+          #!/bin/bash
+          mkdir badger
+          cd badger
+          wget https://github.com/dgraph-io/badger/releases/download/${{ env.BADGER_RELEASE_VERSION }}/badger-checksum-linux-amd64.sha256
+          wget https://github.com/dgraph-io/badger/releases/download/${{ env.BADGER_RELEASE_VERSION }}/badger-linux-amd64.tar.gz
+      - name: Set Dgraph Release Version
+        run: |
+          #!/bin/bash
+          GIT_BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD)
+          if [[ "$GIT_BRANCH_NAME" == "release/v"* ]]; 
+          then
+            echo "this is a release branch"
+          else
+            echo "this is NOT a release branch"
+            exit 1
+          fi
+          DGRAPH_RELEASE_VERSION=$(git rev-parse --abbrev-ref HEAD | sed  's/release\///')
+          echo "making a new release for dgraph "$DGRAPH_RELEASE_VERSION
+          echo "DGRAPH_RELEASE_VERSION=$DGRAPH_RELEASE_VERSION" >> $GITHUB_ENV
+      - name: Make Dgraph Linux Build
+        run: make dgraph DGRAPH_VERSION=${{ env.DGRAPH_RELEASE_VERSION }}
+      - name: Generate SHA for Dgraph Linux Build
+        run: cd dgraph && sha256sum dgraph | cut -c-64 > dgraph-checksum-linux-amd64.sha256
+      - name: Tar Archive for Dgraph Linux Build
+        run: cd dgraph && tar -zcvf dgraph-linux-amd64.tar.gz dgraph
+      - name: Upload Build Artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          path: |
+            badger/badger-checksum-linux-amd64.sha256
+            badger/badger-linux-amd64.tar.gz
+            dgraph/dgraph-checksum-linux-amd64.sha256
+            dgraph/dgraph-linux-amd64.tar.gz
+      - name: Make Dgraph Docker Image
+        run: |
+          make docker-image DGRAPH_VERSION=${{ env.DGRAPH_RELEASE_VERSION }}
+          docker tag dgraph/dgraph:${{ env.DGRAPH_RELEASE_VERSION }} dgraph/dgraph:latest
+      - name: Make Dgraph Standalone Docker Image with Version
+        run: |
+          make docker-image-standalone DGRAPH_VERSION=${{ env.DGRAPH_RELEASE_VERSION }}
+          docker tag dgraph/standalone:${{ env.DGRAPH_RELEASE_VERSION }} dgraph/standalone:latest
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD_TOKEN }}
+      - name: Push Images to DockerHub
+        run: |
+          docker push dgraph/standalone:latest
+          docker push dgraph/standalone:${{ env.DGRAPH_RELEASE_VERSION }}
+          docker push dgraph/dgraph:latest
+          docker push dgraph/dgraph:${{ env.DGRAPH_RELEASE_VERSION }}
```

Comment:
@joshua-goldstein we have a few problems here: 

1.) `latest` is WRONG, if we accidentally run this - we will overwrite the `latest` with an older tag. 
2.) we don't push `slash` image to docker hub*, we push to ECR (from what I know). 

We should address this in a separate PR.

Reply:
Yes that's a good catch...  Perhaps the best solution for now is to just not bring in cd-dgraph.yml?

Reply:
For now I removed cd-dgraph.yml to be on the safe side (we don't want to accidentally clobber the latest tag).  We can add this workflow when it is ready

Reply:
sounds good!